modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -174,5 +188,23 @@ def enum_zones
       logger.debug "Enumerated authoritative dns zones: #{zones}"
       zones
     end
+
+    def enum_records(zone_name, node_name, type)
+      records = []
+      response = execute "/EnumRecords #{zone_name} #{node_name}. /Type #{type}", "EnumRecords", true
+      response.each do |line|
+        line.chomp!
+        logger.debug "Extracting record from dnscmd output '#{line}'"
+        matches = /^@\s+(?<ttl>\d+)\s+(?<type>#{type})\s+(?<record>\S+)$/.match(line)
+        if matches.nil?
+          logger.debug "No DNS record found in this line"
+        else
+          logger.debug "Found record '#{matches['record']}'"
+          records << matches['record'] unless matches.nil?
```

Comment:
@witlessbird  - sorry, misunderstood!

I should really pay more attention to which line specifically is being commented on!  I'll fix this.